src/main.js

```diff
@@ -218,6 +218,22 @@ app.whenReady().then(async () => {
 		})
 	})
 
+	let macDockBounceIds = []
+	ipcMain.on('talk:flashAppIcon', async (event, shouldFlash) => {
+		mainWindow.flashFrame(shouldFlash)
+
+		if (isMac()) {
+			if (shouldFlash) {
+				macDockBounceIds.push(app.dock.bounce())
+			} else {
+				for (const id of macDockBounceIds) {
+					app.dock.cancelBounce(id)
+				}
+				macDockBounceIds = []
+			}
+		}
+	})
```

Comment:
Shouldn't maintain one bounce session ? so 
```suggestion
		if (isMac()) {
			if (macDockBounceId) {
				app.dock.cancelBounce(macDockBounceId)
			}
			if (shouldFlash) {
				macDockBounceId = app.dock.bounce()
			} else {
				macDockBounceId = null
			}
		}
```